rest-api-spec/src/main/resources/rest-api-spec/test/indices.shrink/10_basic.yaml

```diff
@@ -1,26 +1,8 @@
 ---
-setup:
-  # Disable the disk threshold decider so the test doesn't fail if the disk is
-  # > 85% full
-    - do:
-        cluster.put_settings:
-          body:
-            persistent:
-              cluster.routing.allocation.disk.threshold_enabled: false
-          flat_settings: true
-
----
-teardown:
-  # Reset the disk threshold decider so we don't leave anything behind.
-    - do:
-        cluster.put_settings:
-          body:
-            persistent:
-              cluster.routing.allocation.disk.threshold_enabled: null
-          flat_settings: true
-
----
 "Shrink index via API":
+  - skip:
```

Comment:
I don't think we need the skip now though? I'm assuming that you just picked that up in the revert.